SDK_dotnet/Models/ECurrency.cs

```diff
@@ -2,7 +2,9 @@
 {
     public enum ECurrency
     {
+        BTC,
```

Comment:
Don't include BTC yet. In our API client code, we don't have capability to create BTC transactions, so we shouldn't have it right now.